newsfragments/195.feature.rst

```diff
@@ -0,0 +1 @@
+Add support for ``geth v1.14.0``
```

Comment:
Probably another newsfragment for `removal`?

Reply:
yep, this needs to be updated too